backtrader_plotting/bokeh/datatable.py

```diff
@@ -68,7 +67,7 @@ def get_analyzers_tables(self, analyzer: bt.analyzers.Analyzer, strategycls: bt.
             for i, c in enumerate(table_columns):
                 col_name = f'col{i}'
                 cds.add(c[2:], col_name)
-                columns.append(TableColumn(field=col_name, title=c[0], formatter=TableGenerator._get_formatter(c[1])))
+                columns.append(TableColumn(field=col_name, title=c[0], formatter=self._get_formatter(c[1])))

```

Comment:
Since `_get_formatter` is a static method I think it should be called using the class. Why change it to call it through the object? Thank you!

Reply:
@verybadsoldier primarily because in order to get the `number_format` from the `Scheme`, we need `self._scheme`.

Reply:
Yeah sorry, my bad. I didn't see that you turned `_get_formatter` into a member function (and hence added the `self` parameter). Thanks.